spec/models/comment_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  @user = User.create(
+    name: 'Doe', photo: 'https://johndoe.com/me.png',
```

Comment:
- [ ] I noticed that all your tests are not running at all. Please take a look at the image attached and try to fix it 👇 
![image](https://user-images.githubusercontent.com/46839548/197187122-0b2ded45-f157-438b-bf31-ae0017b93c03.png)

Reply:
Sure I will be in contact via slack.